database/guests.ts

```diff
@@ -1,44 +1,55 @@
-declare module globalThis {
-  let guestsDatabase: Guest[];
-}
-
-// Initialize database to empty array only once
-//
-// TODO: Add a full PostgreSQL database connection
-if (!('guestsDatabase' in globalThis)) {
-  globalThis.guestsDatabase = [];
-}
-
-const guests = globalThis.guestsDatabase;
-
-console.log('guests in db', guests);
-
-export type Guest = {
-  id: number;
-  firstName: string;
-  lastName: string;
-  attending: boolean;
-};
-
-export function getGuests() {
+import { Guest } from '../migrations/00000-createTableGuests';
+import { sql } from './connect';
+
+export const getGuestsInsecure = async () => {
+  const guests = await sql<Guest[]>`
+    SELECT
+      *
+    FROM
+      guests
+  `;
   return guests;
-}
-
-export function getGuest(id: number) {
-  return guests.find((guest: Guest) => Number(guest.id) === Number(id));
-}
+};
 
-export function addGuest(guest: Guest) {
-  guests.push(guest);
-  return guests;
-}
+export const getGuestInsecure = async (id: number) => {
+  const [guest] = await sql<Guest[]>`
+    SELECT
+      *
+    FROM
+      guests
+    WHERE
+      id = ${id}
+  `;
+  return guest;
+};
 
-export function deleteGuest(id: number) {
-  const index = guests.findIndex((guest) => guest.id === id);
+export const addGuestInsecure = async (newGuest: Omit<Guest, 'id'>) => {
```

Comment:
Wrong function name. Needs to be `createGuestInsecure` 
https://github.com/upleveled/courses/pull/2546#discussion_r1757148065

Reply:
Changed function name in this commit
[90541d2](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/90541d2eb7827c90daf67259e5483260096536c5)